lib/mapper.js

```diff
@@ -39,12 +60,19 @@ exports.identify = function(identify) {
  */
 
 exports.track = function(track) {
+  var properties = traverse(clone(track.properties()));
```

Comment:
The traverse function cleans up dates (we want this) and the clone function makes a deep copy of the object (we don't need this).